model_training/script.sh

```diff
@@ -37,19 +38,6 @@ rollout_ns_status() {
 	done
 }
 
-function cluster_up() {
```

Comment:
Removing this might impact the instruction in model training in the step prepare cluster. 
https://github.com/sustainable-computing-io/kepler-model-server/tree/main/model_training#pre-step

Could you please also update the instruction to prepare the cluster?